rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -1492,63 +1598,176 @@ void game_list_frame::ShowContextMenu(const QPoint &pos)
 			CreateCPUCaches(gameinfo);
 		}
 	});
-	connect(remove_game, &QAction::triggered, this, [this, current_game, gameinfo, cache_base_dir, name]
+	connect(remove_game, &QAction::triggered, this, [this, current_game, gameinfo, cache_base_dir, hdd1_cache_base_dir, name]
 	{
-		if (current_game.path.empty())
+		if (is_game_running(current_game.serial))
 		{
-			game_list_log.fatal("Cannot remove game. Path is empty");
+			QMessageBox::critical(this, tr("Cannot Remove Game"), tr("The PS3 application is still running, it cannot be removed!"));
 			return;
 		}
 
-		if (is_game_running(current_game.serial))
+		QString text;
+		const bool disc_avail = qstr(current_game.category) == cat::cat_disc_game;
+		const bool disc_auto_detect = disc_avail && (current_game.path.starts_with(rpcs3::utils::get_games_dir()));
```

Comment:
Same here. It should be "is_in_games_dir".
```suggestion
		const bool is_in_games_dir = disc_avail && Emu.IsPathInsideDir(current_game.path, rpcs3::utils::get_games_dir());
```